ansible/roles/monitoring-influxdb/templates/influxdb.retention.j2

```diff
@@ -1,16 +1,16 @@
 {% for database in databases %}
-  influx -execute "alter retention policy default on {{ database }} duration 14d"
+  influx -database "{{ database }}" -username "{{ influx_admin_user }}" -password "{{ influx_admin_password }}" -execute "alter retention policy default on {{ database }} duration 14d"
 
-  influx -execute "drop retention policy _1m on {{ database }}"
-  influx -execute "drop retention policy _10m on {{ database }}"
+  influx -database "{{ database }}" -username "{{ influx_admin_user }}" -password "{{ influx_admin_password }}" -execute "drop retention policy _1month on {{ database }}"
+  influx -database "{{ database }}" -username "{{ influx_admin_user }}" -password "{{ influx_admin_password }}" -execute "drop retention policy _3month on {{ database }}"
 
-  influx -execute "create retention policy  _1m on {{ database }} duration  30d replication 1"
-  influx -execute "create retention policy _10m on {{ database }} duration  90d replication 1"
+  influx -database "{{ database }}" -username "{{ influx_admin_user }}" -password "{{ influx_admin_password }}" -execute "create retention policy _1month on {{ database }} duration  30d replication 1"
```

Comment:
Can you keep the retention policy name consistent with the actual duration - e.g `30d` instead of `_1month`?
Also what's the reason behind the `_`?